docs/platform/concepts/service-integration.rst

```diff
@@ -0,0 +1,39 @@
+Service integration
+===================
+
+Service integrations provide additional functionality and features by connecting different Aiven services together. 
+This includes metrics integration which enables Aiven users to send advanced telemetry data to an `Aiven for PostgreSQL® <https://aiven.io/postgresql>`_ database for metrics and visualize it in `Aiven for Grafana® <https://aiven.io/grafana>`_.
+In addition to the metrics integration, log integration is supported that allows sending logs from any Aiven service to `Aiven for OpenSearch® <https://aiven.io/opensearch>`_.
+
+
+Benefits of integrated telemetry 
+--------------------------------
+
+Aiven automatically provides basic host-level resource metrics (CPU, memory, disk and network) for every service under the **Metrics** tab on the Aiven web console. 
+
+The advanced telemetry feature brings much more detailed, service-specific metrics to the Aiven user, who can then drill down into the service behavior and identify issues and performance bottlenecks.
+
+Frequently asked questions on Aiven service integration
+-------------------------------------------------------
```

Comment:
I'd be very tempted to drop this title, as I don't think it actually adds anything.

I'd then turn the following bolded items into headings - either at the same level by underlining with `-`, or at a sub-level by underlining with `~`. There does seem to be at least some precedent on having such questions in some of the other HowTo articles.

Reply:
Thanks Tibs. https://github.com/aiven/devportal/pull/1573/commits/29b64324e5dacf794fa2a133e9162a3e0e7a4cb1 takes care of that.